<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltersDialog</class>
 <widget class="QDialog" name="FiltersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="callSignText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="callSignCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CallSign</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="AddressCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackNumberText">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="trackNumberCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Track Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="categoryCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetFiltersButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Filters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FiltersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FiltersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
